ProjFS.Mac/PrjFSLib/PrjFSLib.cpp

```diff
@@ -1035,10 +1041,16 @@ static bool InitializeEmptyPlaceholder(const char* fullPath, TPlaceholder* data,
         data->header.formatVersion = PlaceholderFormatVersion;
         
         static_assert(is_pod<TPlaceholder>(), "TPlaceholder must be a POD struct");
-        if (AddXAttr(fullPath, xattrName, data, sizeof(TPlaceholder)))
+        
+        errno_t result = AddXAttr(fullPath, xattrName, data, sizeof(TPlaceholder));
+        if (0 == result)
```

Comment:
I'm happy with this as is.

If we wanted to go one step further we could define SUCCESS=0, and then check against that but I have no trouble understanding what's going on.

Reply:
> If we wanted to go one step further we could define SUCCESS=0, and then check against that but I have no trouble understanding what's going on.

It's unfortunate there's not already something defined for success being 0.  I thought about adding something but didn't want it to inadvertently get used in the wrong spot.  I think I'll leave this as-is for now and we can change it later if the 0s get too unwieldily.